ci-operator/jobs/infra-periodics.yaml

```diff
@@ -308,6 +308,41 @@ periodics:
     - name: token
       secret:
         secretName: github-credentials-openshift-bot
+- agent: kubernetes
+  labels:
+    ci.openshift.io/role: infra
+  extra_refs:
+  - base_ref: master
+    org: openshift
+    repo: release
+  cron: ""
+  decorate: true
+  interval: 12h
+  name: periodic-prow-auto-config-brancher
+  spec:
+    containers:
+    - command: ["/usr/bin/autoconfigbrancher"]
+      args:
+      - --github-token=/etc/github/oauth
```

Comment:
Do you not need the github endpoints?

Reply:
In the implementation it uses the default endpoints for v3/v4 github api.
https://github.com/openshift/ci-tools/blob/master/cmd/autoconfigbrancher/main.go#L139
Did not use proxy at all. This is also the case for autoowners and autobumper.

Will fix.

Reply:
https://github.com/openshift/ci-tools/pull/172

Reply:
/hold

Reply:
Done.